ui/apps/vibes/src/utils/format-date.ts

```diff
@@ -0,0 +1,11 @@
+import dayjs from 'dayjs';
```

Comment:
you can add this to the time.ts in design-system-core, so we have all the date formatting in one place